hbp_knowledge/proteostasis/niforou2014.bel

```diff
@@ -243,72 +278,95 @@ phase II and antioxidant enzymes including NAD(P)H quinone
 oxidoreductase 1 (Nqo1), heme oxygenase 1 (Hmox1), glutamane-
 cysteine ligase (GCL) and glutathione S transferases (GSTs)
 [84,85,87,88]."
+SET Confidence = "Medium"
 bp(GO:"response to oxidative stress") -| act(p(HGNC:KEAP1))
 act(p(HGNC:KEAP1)) -> deg(p(HGNC:NFE2L2))
-bp(GO:"response to oxidative stress") -> p(HGNC:NFE2L2)
+bp(GO:"response to oxidative stress") -> p(HGNC:NFE2L2, loc(MESH:"Cell Nucleus"))
 complex(p(HGNC:NFE2L2),p(HGNC:MAF)) -> p(HGNC:NQO1)
 complex(p(HGNC:NFE2L2),p(HGNC:MAF)) -> p(HGNC:HMOX1)
 complex(p(HGNC:NFE2L2),p(HGNC:MAF)) -> p(HGNC:GCLC)
 complex(p(HGNC:NFE2L2),p(HGNC:MAF)) -> p(FPLX:GST)
+UNSET Confidence
 
 SET Evidence = " Moreover, Nrf2 contributes to cellular proteostasis
 by regulating the expression of molecular chaperones [89], as well
 as of additional players of proteome stability and maintenance,
 namely the proteasome subunits [90–92]."
+SET Confidence = "High"
 p(HGNC:NFE2L2) -> bp(HBP:Proteostasis)
+UNSET Confidence
 
 SET Evidence = "Proteome oxidation and
 instability has been associated with ageing and the progression of
 several age-related diseases, including cardiovascular disorders, neu-
 rodegeneration, and cancer [7,95,96]."
-a(CHEBI:radical) -> bp(GO:"protein oxidation")
+SET Confidence = "Medium"
+#refernce for the comment statement?
+#a(CHEBI:radical) -> bp(GO:"protein oxidation")
 bp(GO:"protein oxidation") -- bp(GO:aging)
 bp(GO:"protein oxidation") -- path(MESH:"Cardiovascular Diseases")
 bp(GO:"protein oxidation") -- bp(HP:"Neurodegeneration")
 bp(GO:"protein oxidation") -- path(MESH:Neoplasms)
+UNSET Confidence
 
+#Cannot understand the relation of BEL statement with evidence statement
```

Comment:
There was no term such as oxidants, but in this paper the oxidants under consideration were the radicals ( as the are able to remove electrons from other proteins ).
So I just coded this with the term "radicals", which can be seen as equal as oxidant agent in this specific case